### DISCUSSION TITLE
cargo-shuttle install error on mac

### DISCUSSION THREAD

**Discussion by dev-puzzler** [shuttle-hq/shuttle#1163] [Q&A]
<img width="1352" alt="스크린샷 2023-08-20 오전 12 21 24" src="https://github.com/shuttle-hq/shuttle/assets/142076054/eccb9480-69e5-4ae0-b296-194c8df4658c">

how to fix?

**Comment by oddgrd** [comment]
Hey @dev-puzzler, the referenced methods are new in Rust 1.70, so you'll need to update your version of Rust: `rustup update stable`.
